### PR TITLE
Added feature for nodate in filename

### LINKED ISSUE #7: Option for no Datetime on end of file
I want to pick up this file at a destination with the same name on a scheduled basis. Having a datetime doesn't help me accomplish that. 

Plan is to make the datetime be configurable

### PR DESCRIPTION
**Pull request by visch**
Closes #7 

# Description of change
Added flag for filename_include_date. Maybe there's a slicker way to do this?

# Manual QA steps
Ran this locally multiple times against the same files. 
 
# Risks
If the Schema for a stream is emitted multiple times the file will not contain all records sent 
 
# Rollback steps
 - revert this branch

### CODE REVIEW

**Comment by hsyyid** [comment]
Thanks for the contribution. A few questions: in my local testing with `target-csv` I am pretty sure a datetime is emitted based on the `VERSION` emitted by the tap. Seems like your use case here is emitting the files with the **current** `date`, is that correct?

**Comment by hsyyid** [comment]
Thanks for the contribution! Will review locally.

File: target_csv.py [hunk 1]
```diff
@@ -38,13 +38,13 @@ def flatten(d, parent_key='', sep='__'):
     return dict(items)
 
 
-def persist_messages(delimiter, quotechar, messages, destination_path, fixed_headers):
+def persist_messages(delimiter, quotechar, messages, destination_path, fixed_headers, filename_include_date):
     state = None
     schemas = {}
+    streamname_to_filename = {}
```
Comment (visch) [inline target_csv.py:44]: This isn't being populated. Looks like I pushed up the wrong commit here. I'll close this PR.

File: target_csv.py [hunk 2]
```diff
@@ -61,8 +61,7 @@ def persist_messages(delimiter, quotechar, messages, destination_path, fixed_hea
 
             validators[o['stream']].validate(o['record'])
 
-            filename = o['stream'] + '-' + now + '.csv'
-            filename = os.path.expanduser(os.path.join(destination_path, filename))
+            filename = generate_filename(filename_include_date, destination_path, o['stream'], ".csv", now)
```
Comment (visch) [inline target_csv.py:64]: This won't work when filename_include_date is False. Pretty simple fix but I"ll close the PR for now as this doesn't work for everyone

**Pull request closed without merging**